packages/core-js-sdk/src/httpClient/helpers/index.ts

```diff
@@ -1 +1,14 @@
 export { default as createFetchLogger } from './createFetchLogger';
+
+export function transformSetCookie(setCookieHeader: string) {
+  // Split the header by semicolons to separate different attributes
+  var cookiesString = setCookieHeader.split(';');
+
+  return cookiesString.reduce((acc, cookie) => {
+    const [key, value] = cookie.split('=');
```

Comment:
@nirgur FYI I just had an interesting issue (with query params) that their value contains `=`

this means, that if the value of the cookie will contain `=`, the cookies won't get it
there, I did something quite naive to catch it
```
       const indexOfEquals = part.indexOf('=');
        const key = part.slice(0, indexOfEquals);
        const value = part.slice(indexOfEquals + 1);
        ```
        
        consider adding something similar here